laghos.cpp

```diff
@@ -784,6 +784,91 @@ int main(int argc, char *argv[])
     romOptions.restore = rom_restore;
     romOptions.offsetType = getOffsetStyle(offsetType);
 
+
+    std::string offlineParam_outputPath = outputPath + "/offline_param.csv";
+    if (rom_offline)
+    {
+        MFEM_VERIFY(romOptions.parameterID >= 0 || romOptions.offsetType != interpolateOffset, "-rostype interpolate is not compatible with non-parametric ROM.");
```

Comment:
Several lines among 791-798 are quite long and should be shortened (based on the usual style of mfem and laghos).